src/vanilla/Templates/PackageJson.cshtml

```diff
@@ -34,6 +43,6 @@
     "url": "@(Model.BugsUrl)"
   },
   "scripts": {
-    "build": "tsc && webpack"
+    "build": "tsc @(string.IsNullOrEmpty(Model.DefaultApiVersion) ? "" : "-b") && webpack"

```

Comment:
What does "-b" do? It's not listed on the [TypeScript Compiler Options](https://www.typescriptlang.org/docs/handbook/compiler-options.html).

Reply:
It stands for --bleeding-edge (no, it's short for [--build](https://www.typescriptlang.org/docs/handbook/project-references.html), and it really should be documented.)